packages/camera/camera/ios/Classes/FLTCam.m

```diff
@@ -355,10 +362,15 @@ - (void)captureOutput:(AVCaptureOutput *)output
   if (output == _captureVideoOutput) {
     CVPixelBufferRef newBuffer = CMSampleBufferGetImageBuffer(sampleBuffer);
     CFRetain(newBuffer);
-    CVPixelBufferRef old = _latestPixelBuffer;
-    while (!OSAtomicCompareAndSwapPtrBarrier(old, newBuffer, (void **)&_latestPixelBuffer)) {
-      old = _latestPixelBuffer;
-    }
+
+    __block CVPixelBufferRef old = nil;
```

Comment:
Please give this a more meaningful name while you're updating this code (e.g., `previousPixelBuffer`).